Proposals/BTC Forks Roadmap Proposal.md

```diff
@@ -181,15 +131,31 @@ The main goal of the first bitcoin spinoff will be to allow bitcoin to scale saf
 
 ---
 
-###7. Future Objectives
+
+##6. Evaluation
+
+*How to measure your results and effectiveness. This should correspond to your objectives.*
+
+
+##7. Rough Schedule
+
+**Months 1-3 :** 
+Community development, establishment of necessary channels, build, and testing infrastructure, and community release of several proposed forks (including multiple client bases and PoW implementations). Our goal is to create a collaborative environment in which forks can be proposed, tested, evaluated, and deployed by the community and marketplace. We will research key challenges, like PoW changes, difficulty adjustment, script limits, and effects of blocksize cap through tests on our public test network.
+
+**Months 3-6 :** 
+Client release of final binaries, and establishment of exchange relationships necessary to begin trading of dominant fork. Deployment of required network infrastructures, including an initial node and mining network. We will focus on getting a fork ready for widespread use, getting a working product to the marketplace.
+
+**Months 6+ :**
+Support of the initial and any further forks through the creation of community resources for comparison, discussion, and decision making. We will faciliatate the overall Bitcoin governance process by encouraging sane forks, regardless of whether or not they are contentious. We will aim to provide the market with all resources and information necessary to make informed decisions.
+
+
+##8. Future Objectives
```

Comment:
I would add some more specific objectives for the future. These may be started in parallel to the MVF implementation and build-up of infrastructure to support fork development processes:

(NOTE: Github's markdown renumbered them - the intent is to add on to the list below, i.e. make them points 3, 4, 5)

3. Work on an objective description of the fundamental principles that underlie Bitcoin's evolution as a technology.
This will cover the CALs etc.

4. Work on elaborating a (subjective) set of constitutional principles that the community supporting the solution "Bitcoin: A Peer-to-Peer Electronic Cash Network for payments and settlement transactions" can agree on. This will require conducting a constitution-forming process, starting from the already existing inputs (Satoshi's confirmed writings, principles extolled at Satoshi's Roundtable workshop, etc.)

5. Work with industry to develop a Bitcoin Industry Code of Conduct or Ethical Code that lines up with the constitutional principles, but puts them in practical context. This document can provide guidance on conduct and best practices for miners, pools, exchanges, software development houses, academic researchers etc.